searchcore/src/vespa/searchcore/proton/server/storeonlyfeedview.cpp

```diff
@@ -142,46 +143,45 @@ std::vector<document::GlobalId> getGidsToRemove(const IDocumentMetaStore &metaSt
     return gids;
 }
 
-void putMetaData(documentmetastore::IStore &meta_store, const DocumentId &doc_id,
+void putMetaData(documentmetastore::IStore &meta_store, const GlobalId & gid, const DocumentId & doc_id,
```

Comment:
This change seems to be unneeded.

Reply:
Agree, since the branch will never be taken.